components/MentorshipsList/MentorshipInfo.js

```diff
@@ -12,6 +12,16 @@ const markdownTheme = {
 
 const MentorshipInfo = ({ mentorship }) => {
   const { message, mentorshipDate, status, longterm } = mentorship;
+  const date = new Date(mentorshipDate);
+  const year = date.getUTCFullYear();
+  const month = date.getUTCMonth();
+  const day = date.getUTCDate();
+  const hours = date.getUTCHours();
+  const minutes = date.getUTCMinutes();
+  const seconds = date.getUTCSeconds();
+  const dateInUTC = new Date(Date.UTC(year, month, day, hours, minutes, seconds));
+  const formattedDateTime = new Intl.DateTimeFormat('en-GB', { dateStyle: 'medium', timeStyle: 'short' }).format(dateInUTC);
```

Comment:
The stored date _should_ be already in UTC format, but I now see that there are some date fields in the xata db that are `string` and some others that are `datetime`.

I think there is no need to extract one by one all the components and then use them to build a `new Date` because we could re-use the first `const date = new Date(mentorshipDate);`.

Also, on second toughts, to simplify, let's skip the time completely, so that we avoid the timezone problem.

I suggest the following change:

```suggestion
  const dateInUTC = new Date(mentorshipDate);
  const formattedDateTime = new Intl.DateTimeFormat('en-GB', { dateStyle: 'medium' }).format(dateInUTC);
```

What do you think?

Reply:
I had tried something simple along the lines of what you recommended, I definitely didn't start it out this complicated. But I kept getting errors every time I would run it which is why I ended up doing this long complicated thing. I probably wasn't paying attention to how I was doing it so maybe I was just doing something wrong. But I'll go back to making it simple.

Also, if we're letting go of the time, then I'll just roll back to the second commit because that one was the date only.

Reply:
That's great! Thanks for taking care of this, and sorry to add more work on it!

Reply:
No, it's fine. It's not that much work at all. Let me get on that now.

Reply:
I had awful internet so it took forever to roll back, but it should be fine now and just show the date

Reply:
I had awful internet so it took forever to roll back, but it should be fine now and just show the date

Reply:
thanks a lot :)